File: articles/leadership-in-organizational-psychology/fortran/leadership_capacity_score.f90
program leadership_capacity_score_program
  implicit none
  real :: score
  score = 0.11*83.0 + 0.11*81.0 + 0.10*79.0 + 0.11*82.0 + 0.10*80.0 + &
          0.10*81.0 + 0.08*80.0 + 0.09*82.0 + 0.08*81.0 + 0.08*80.0 - &
          0.07*20.0 - 0.08*18.0 - 0.07*22.0 - 0.06*24.0
  print *, "Synthetic leadership capacity score:", score
  print *, "Responsible-use reminder: institutional learning only."
end program leadership_capacity_score_program

File: articles/leadership-in-organizational-psychology/rust/leadership_capacity_score.rs
fn leadership_capacity_score(values: [f64; 14]) -> f64 {
    let weights = [0.11,0.11,0.10,0.11,0.10,0.10,0.08,0.09,0.08,0.08,-0.07,-0.08,-0.07,-0.06];
    values.iter().zip(weights.iter()).map(|(v,w)| v*w).sum()
}

fn main() {
    let score = leadership_capacity_score([83.0,81.0,79.0,82.0,80.0,81.0,80.0,82.0,81.0,80.0,20.0,18.0,22.0,24.0]);
    println!("Synthetic leadership capacity score: {:.2}", score);
    println!("Responsible-use reminder: institutional learning only.");
}
